content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -0,0 +1,464 @@
+
+## Introduction
+
+In this tutorial we'll be showing you how you might go about converting an existing gatsby-mdx blog to use TinaCMS. We've created a starter repo for you to follow along with. The repo is a fork of the official [Gatsby md blog starter](https://www.gatsbyjs.com/starters/gatsbyjs/gatsby-starter-blog).
+
+## Limitations
+
+There are a few limitations with the approach outlined in this guide.
+
+- You don't get visual editing
+- You lose out in Gatsby's image optimization
+- Gatsby uses Github Flavoured Markdown, which TinaCMS doesn't fully support
+
+## Getting started
+
+First you'll want to make sure you have the correct version of node installed. We'll be using the node version specified in the repo's `.nvmrc` file, v20.12.0. You can run the code below to make sure you have the correct version installed.
+
+```powershell
+node --version
+```
+
+Once you've made sure you have the right version of node installed, clone the project. Then navigate into the blog's directory.
+
+```powershell
+git clone https://github.com/tinacms/gatsby-mdx-example-blog
+cd gatsby-mdx-example-blog
+```
+
+## Adding Tina
+
+Awesome! You're all set up and ready start adding TinaCMS. You can initialize it using the command below.
+
+```powershell
+npx @tinacms/cli@latest init
+```
+
+After running the command above you'll receive a few propmts
+
+1. When prompted to select a framework select `“other”`
+2. Choose `“NPM”` as your package manager
+3. When asked if you'd like to use Typescript choose `“no”`
+4. Set the public assets location to `“public”`
+
+## Setting up Gatsby for Tina
+
+Now that we've added Tina to our project there's a few more steps we need to do to get Tina working with Gatsby. Add the following line at the top of `tina/config.js`
+
+```diff
+export default defineConfig({
++  client: { skip: true },
+// ...
+```
+
+Then add the following line in `gatsby-node.js`
+
+```diff
++ const express = require("express");
++ exports.onCreateDevServer = ({ app }) => {
++   app.use("/admin", express.static("public/admin"));
++ };
+```
+
+To save ourself some pain we can also change our startup command to make sure Tina gets run when we run our app in development mode.
+
+```diff
+  "scripts": {
+    "build": "gatsby build",
+-   "develop": "gatsby develop",
++   "develop": "npx tinacms dev -c \"gatsby develop\"",
+    "format": "prettier --write \"**/*.{js,jsx,ts,tsx,json,md}\"",
+    "start": "gatsby develop",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "test": "echo \"Write tests! -> https://gatsby.dev/unit-testing\" && exit 1"
+  }
+```
+
+## Configuring our Schema
+
+Now we've added Tina to our project but it's not pointing at our existing markdown files **(Oh noes!)**. Let fix that. Open up `tina/config.ts` and change the path to point to our blog directory. We'll also need to update our schema since we'll be working with mdx files.
```

Comment:
```suggestion
Now we've added Tina to our project but it's not pointing at our existing markdown files. Let fix that. 

Open up `tina/config.ts` and change the path to point to our blog directory. We'll also need to update our schema since we'll be working with mdx files.
```

Split up the paragraph and remove the oh no I reckon 😆

Reply:
Committed 👍